department-of-veterans-affairs/caseflow
# Improve legibility of Hearing Date dropdown

Resolves [CASEFLOW-1491](https://vajira.max.gov/browse/CASEFLOW-1491)

### Description
Changes the display of the hearing time dropdown

### Acceptance Criteria

### Testing Plan
1. Shadow user `BVASYELLOW`: http://localhost:3000/test/users
2. Navigate to the Schedule Veterans age: http://localhost:3000/hearings/schedule/assign
3. Choose "St. Petersburg, FL" from the list and go to the AMA waiting tab
4. Choose one of the veterans waiting and once on the Case Details page, click the "Schedule Veteran" action
5. Wait for the hearing dates to be populated and then check that the display matches the mockup in the ticket
6. Continue scheduling and ensure there are no errors

### User Facing Changes

 BEFORE|AFTER
 ---|---
<img width="558" alt="Screen Shot 2021-05-27 at 11 52 29 AM" src="https://user-images.githubusercontent.com/61989526/119885971-5522d100-bee7-11eb-81c0-252738369aea.png">|<img width="597" alt="Screen Shot 2021-05-27 at 12 26 03 PM" src="https://user-images.githubusercontent.com/61989526/119885981-58b65800-bee7-11eb-8212-cf9981e95b7c.png">


### Code Documentation Updates
